src/chains/ethereum/block/src/runtime-block.ts

```diff
@@ -113,15 +131,11 @@ export class RuntimeBlock {
       gasLimit: gasLimit.toBigInt(),
       gasUsed: gasUsed.toBigInt(),
       timestamp: timestamp.toBigInt(),
-      baseFeePerGas: baseFeePerGas ?? 0n,
+      baseFeePerGas: baseFeePerGas ?? undefined,
```

Comment:
We just set `baseFeePerGas` to undefined here in the `header`, instead of setting it to `0` and adding a `serializeBaseFeePerGas` flag (below).

Reply:
Isn't `baseFeePerGas` going to be `undefined` anyways?
```suggestion
      baseFeePerGas: baseFeePerGas
```